bigbluebutton-html5/package.json

```diff
@@ -39,6 +39,7 @@
     "react-modal": "~1.7.7",
     "react-router": "~3.0.2",
     "react-tabs": "~1.0.0",
+    "react-toastify": "^2.1.0",
```

Comment:
This package now breaks the client, because the author has updated it wrongfully. It does not match our react version anymore. Maybe fix this version to just `2.1.0`